### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by koppor**
Enjoy!

### CODE REVIEW

File: de/BaseFrame.md [hunk 1]
```diff
@@ -1,43 +1,41 @@
...
+# JabRef's main window
 
-# Das Hauptfenster von JabRef
+*Note:* most menu actions referred in the following have keyboard shortcuts, and many are available from the toolbar. The keyboard shortcuts are found in the pull-down menus.
 
-*Anmerkung:* Die meisten Menüfunktionen, auf die im Folgenden hingewiesen wird, haben Tastenkürzel, und viele Funktionen können über die Werkzeugleiste aufgerufen werden.
+This is the main window from where you work with your databases. Below the menubar and the toolbar is a tabbed pane containing a panel for each of your currently open databases. When you select one of these panels, a table appears, listing all the database's entries, as well as a configurable selection of their fields.
 
-Dies ist das Hauptfenster, in dem Sie mit Ihrer Datei arbeiten. Unter der Menüleiste und der Werkzeugleiste befindet sich die Tableiste mit Tabs (Reitern) für jede Ihrer geöffneten Dateien. Wenn Sie einen dieser Tabs anklicken, erscheint eine Tabelle, die alle Datensätze und einige der Datenfelder auflistet.
+- You decide which fields are shown in the table by checking the fields you want to see in the **Preferences** dialog.
```
Comment (Siedlerchr) [inline de/BaseFrame.md]: There already exists a German translation which will now be overwritten!

**Comment by koppor** [comment]
I know. I am currently in contact with the support.

File: de/ACMPortal.md [hunk 2]
```diff
@@ -1,4 +1,15 @@
 ---
-redirect:   /en/ACMPortal
-layout:     redirect
+title: Fetching entries from ACM Portal
```
Comment (tobiasdiez) [inline de/ACMPortal.md]: Is it normal that the untranslated page is replaced by the english text? A redirect has the huge advantage that we do not need to maintain 20 english versions (or is this handled automatically by the tool?)

**Comment by koppor** [comment]
Automatically handled. Currently no idea how to handle that. Maybe it is
impossible...

Am 29.11.2017 12:53 schrieb "Tobias Diez" <notifications@github.com>:

**Pull request closed without merging**